lib/storage.js

```diff
@@ -22,21 +22,15 @@ const path = require('path');
 
 const DATA_PROTOCOL = 'data:';
 
-function isValidLocalFile(fileName, basePath){
-    const fullPath = path.join(basePath, fileName);
-    if ( (!fullPath.startsWith("/in")) || fullPath.endsWith("/..") || fullPath.includes("/../")) {
-        return false;
-    }
-    return fs.existsSync(fullPath);
-}
-
 async function getSource(paramsSource, inDirectory, disableSourceDownload) {
     // Note: validation has happened in validate.js before
 
     if (process.env.WORKER_TEST_MODE) {
         // local file support for `Asset Compute test-worker` unit tests
         // not supported for clients in production
-        if (!isValidLocalFile(paramsSource.url, inDirectory)) {
```

Comment:
I fail to recollect, why we replaced `isValidLocalFile` with `!fs.existsSync(filePath)`

Reply:
It's actually something I did just last week because of the `/in` and `/out` changes: instead of reading/writing directly to those folders that are mounted in the container (and which lead to problems), we copy instead. This also meant we had to adjust this validation logic and build the right path: `/nodejsAction/xyz/in/` plus `file.jpg`. The input params set by `test-worker` still have just the filename in them, e.g. `file.jpg`, so this made it easy.

See commit https://github.com/adobe/asset-compute-sdk/pull/22/commits/ea0ad47fb6de5e5fb3c5e31bbffada328cd7e267